### ISSUE TITLE
GL error 1282 invalid operation on Linux / macOS

### ISSUE THREAD

**Issue by thorn0906** [armory3d/armory#848]
Both Krom and C++ generate the following errors (WebGL doesn't work either, but for different reasons):
```
GL Error 1282 /home/lubos/Downloads/Krom/Kore/Backends/Graphics4/OpenGL/Sources/Kore/OpenGL.cpp 205

GL Error 1282 /home/lubos/Downloads/Krom/Kore/Backends/Graphics4/OpenGL/Sources/Kore/PipelineStateImpl.cpp 196

GL Error 1282 /home/lubos/Downloads/Krom/Kore/Backends/Graphics4/OpenGL/Sources/Kore/PipelineStateImpl.cpp 199

GL Error 1282 /home/lubos/Downloads/Krom/Kore/Backends/Graphics4/OpenGL/Sources/Kore/OpenGL.cpp 241
```
This happens for any scene. I'm using Armory 0.4 on Void Linux, with an Radeon R9 270x GPU, and my headers say 1282 corresponds to GL_INVALID_OPERATION. A quick look at the paths mentioned in the error message seems to suggest that the error is occurring on calls to glUniform*.

**Comment by rivenblades** [comment]
Will this get solved in armory 0.5 also?

**Comment by luboslenco** [comment]
Fixed `GL Error 1282` at https://github.com/armory3d/armory/commit/89afc182e8356cef7eac97551215ab652c63a2ff.
